components/Main/style.css

```diff
@@ -50,6 +132,36 @@
     padding: 25px 25px 5px;
   }
 }
+.green-true-icon {
+  height: 20px;
+  width: 20px;
+  cursor: pointer;
+}
+.red-delete-icon {
+  height: 20px;
+  width: 20px;
+  cursor: pointer;
+}
+.edit-project-name-input {
+  position: relative;
+  right: 11px;
+  top: -0.5px;
+  border-color: rgba(112, 112, 112, 0.2);
+  border-width: 1px;
+  border-style: solid;
+  border-radius: 5px;
+  padding-left: 10px;
+  background: #ffffff;
+  min-width: 200px;
+  max-width: 500px;
+  height: 38px;
+
+  font-size: 1.4rem;
+  font-weight: 500;
+
+  /* max-width: 500px; */
+  word-wrap: break-word;
+}
 

```

Comment:
### General CSS Suggestions:

1. **Consistent Naming Conventions:**
     The classes like `.three-dots-icon`, `.menu-container`, and .`actions-container` are well-named for understanding purposes. Ensure this naming is consistent throughout the project for easy maintenance.

2. **Separate .show and .hide Classes for Consistency:**
    Instead of toggling between `display: none` and `display: inline`, use `display: flex` on `.show` where flexbox layout is applied, and `display: none` on `.hide`.

3. **Improving Readability with Comments:**
    Adding comments to define sections within the CSS (e.g., “/* Icon styles /” and “/ Actions container styles */”) will make the file easier to navigate.
    
4. **Avoid Inline Style Overrides in CSS:** 
    For example, `.edit-project-name-input` has `inline max-width: 500px;` commented out. Setting consistent limits in CSS with media queries where necessary will ensure this styling works across screen sizes.